integration-tests/zendeskRequestAPI.spec.ts

```diff
@@ -21,30 +94,39 @@ describe('Zendesk request integrity', () => {
     }
 
     const headers = {
-      'X-Zendesk-Webhook-Signature': invalidSignature
+      ...generateSignatureHeaders(webhookRequestData)
     }
 
-    const invalidRequestError = async () => {
-      return axios({
-        url: getEnvVariable('AWS_WEBHOOK_URL'),
-        method: 'POST',
-        headers,
-        data
-      })
-        .then(() => {
-          throw 'Gateway did not return an error'
-        })
-        .catch((error) => {
-          if (error.response) {
-            return error.response
-          }
-          console.log(error)
-          throw 'No response recieved from gateway'
-        })
+    const errorResponse = await invalidRequestError(headers, webhookRequestData)
+    expect(errorResponse.status).toEqual(404)
+    expect(errorResponse.data.message).toEqual('Zendesk ticket not found')
+  })
+
+  test('API Gateway returns a 400 response if the request does not match info in corresponding Zendesk ticket', async () => {
+    const webhookRequestData = {
+      //ticket with ID 881 exists with different info
+      zendeskId: '881',
```

Comment:
Could we maybe call the function to raise a ticket and use the id, rather than hardcode. The ticket could be deleted by anybody even outside our team. I'll add something to delete tickets shortly, so it can be used for teardown.

Reply:
I'd rather not extend this PR any further (it's been open a long time now), so would look to pick that up separately.

Reply:
I agree with @rikumolugds . The code to create a ticket is already there in the codebase, so could you try just adding it and seeing if it works? Shouldn't be too big a job, and I agree that hard-coding something that could be removed at any point seems like a bit of a bad idea.